packages/fluentui/react-northstar/src/themes/teams/types.ts

```diff
@@ -107,7 +107,7 @@ export type TeamsThemeStylesProps = {
   Grid: GridProps;
   Header: HeaderProps;
   HeaderDescription: HeaderDescriptionProps;
-  Icon: IconStylesProps;
+  Icon: SvgIconStylesProps;
```

Comment:
This looks weird. Should it be `SvgIcon`?

```suggestion
  SvgIcon: SvgIconStylesProps;
```